www/includes/easyparliament/templates/html/section/index.php

```diff
@@ -0,0 +1,19 @@
+<div class="full-page__row">
```

Comment:
I think something more needs to be done for /debates and /written-answers-and-statements. As it is, it's not clear to me that there is more equivalent data underneath the search box or that the page has anything more than the first thing (Commons debates or Written answers) on it.